docs/develop/gettingstarted.rst

```diff
@@ -100,7 +100,7 @@ For the stm32 port, the ARM cross-compiler is required:
 
 .. code-block:: bash
 
-   $ sudo apt-get install arm-none-eabi-gcc arm-none-eabi-binutils arm-none-eabi-newlib
+   $ sudo apt-get install gcc-arm-none-eabi binutils-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
```

Comment:
I think this is enough for most users. `gcc-arm-none-eabi` depends on `binutils-arm-none-eabi` so it is redundant to add it to the list. `libnewlib-arm-none-eabi` is recommended `gcc-arm-none-eabi` so gets installed with the default Ubuntu config too (and this probably isn't needed anyway since we don't used the standard C library). And most people probably aren't going to have C++ code either.

```suggestion
   $ sudo apt-get install gcc-arm-none-eabi
```

Reply:
Is this a "I think this will work" or a "I tried this and it works"? Because mine was the latter. It may be excessive but it fixes incorrect instruction.

Reply:
I have only ever manually installed `gcc-arm-none-eabi` and not the other packages (the rest are installed automatically as dependencies).

```
$ apt-mark showmanual | grep arm-none-eabi
gcc-arm-none-eabi
$ apt-mark showauto | grep arm-none-eabi
binutils-arm-none-eabi
libnewlib-arm-none-eabi
libstdc++-arm-none-eabi-dev
libstdc++-arm-none-eabi-newlib
```

Reply:
The CI here runs Ubuntu and does:
```
sudo apt-get install gcc-arm-none-eabi libnewlib-arm-none-eabi
```
It builds C++ code via the user C modules examples.  So this is probably what we should write in the docs.